### ISSUE TITLE
Is it possible to isolate the prompting logic and decouple it from showing the rating alert?

### ISSUE THREAD

**Issue by mpprdev** [nicklockwood/iRate#207]
This way we can reuse the prompting logic to display other information.

**Comment by nicklockwood** [comment]
Yes, it's possible by using the delegate. But have you looked at my iPrompt and iNotify libraries? They might be better suited for what you're trying to do.
